test/e2e/operator.go

```diff
@@ -98,17 +98,15 @@ var _ = Describe("ARO Operator - Internet checking", func() {
 			return err
 		})
 		Expect(err).NotTo(HaveOccurred())
-	})
-	It("sets InternetReachableFromMaster to true when the default URL is reachable from master nodes", func(ctx context.Context) {
```

Comment:
Is it intentional that we are removing these tests?

Reply:
The two tests are replaced by these two lines
```g.Expect(conditions.IsTrue(co.Status.Conditions, arov1alpha1.InternetReachableFromMaster)).To(BeTrue())
			g.Expect(conditions.IsTrue(co.Status.Conditions, arov1alpha1.InternetReachableFromWorker)).To(BeTrue())

Reply:
Okay so as opposed to a separate test, we are going to wait for these conditions to be true after each test. I understand.